wurst/data/LinkedListIndexedModule.wurst

```diff
@@ -0,0 +1,130 @@
+package LinkedListIndexedModule
+import HashMap
+
+/** Turns a class into a collection of linked lists where each list
+    is composed of instances of the same index. Each instance 
+    knows it's previous and next member in its index's list. */
```

Comment:
Doesn't answer "Why?". What's this package useful for?

Reply:
To modularize stuff we are already doing in the stdlib.